tests/SoCreate.Extensions.Caching.Tests/SoCreate.Extensions.Caching.Tests.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="AutoFixture.AutoMoq" Version="4.11.0" />
     <PackageReference Include="AutoFixture.Xunit2" Version="4.11.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
-    <PackageReference Include="Microsoft.ServiceFabric" Version="7.0.*" />
+    <PackageReference Include="Microsoft.ServiceFabric" Version="7.1.409" />
```

Comment:
This should be changed to 7.1.*

Reply:
I had it as 7.1.* before, but there was a version conflict still. I'll get the error message.

Reply:
Nevermind, 7.1.* is fine to do now...before it was acting like 7.1.* resolved a different version of Microsoft.ServiceFabric than what another dependency was using.